share/functions/__fish_config_interactive.fish

```diff
@@ -261,63 +261,9 @@ function __fish_config_interactive -d "Initializations that should be performed
-        # For our purposes, we use both.
-        set -l os
-        if test -r /etc/os-release
-            set os (string match -r '^ID(?:_LIKE)?\s*=.*' < /etc/os-release | \
-            string replace -r '^ID(?:_LIKE)?\s*=(.*)' '$1' | string trim -c '\'"' | string split " ")
-        end
-
-        # First check if we are on OpenSUSE since SUSE's handler has no options
-        # but the same name and path as Ubuntu's.
-        if contains -- suse $os || contains -- sles $os && type -q command-not-found
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                /usr/bin/command-not-found $argv[1]
-            end
-            # Check for Fedora's handler
-        else if test -f /usr/libexec/pk-command-not-found
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                /usr/libexec/pk-command-not-found $argv[1]
-            end
-            # Check in /usr/lib, this is where modern Ubuntus place this command
-        else if test -f /usr/lib/command-not-found
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                /usr/lib/command-not-found -- $argv[1]
-            end
-            # Check for NixOS handler
-        else if test -f /run/current-system/sw/bin/command-not-found
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                /run/current-system/sw/bin/command-not-found $argv
-            end
-            # Ubuntu Feisty places this command in the regular path instead
-        else if type -q command-not-found
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                command-not-found -- $argv[1]
-            end
-            # pkgfile is an optional, but official, package on Arch Linux
-            # it ships with example handlers for bash and zsh, so we'll follow that format
-        else if type -p -q pkgfile
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                set -l __packages (pkgfile --binaries --verbose -- $argv[1] 2>/dev/null)
-                if test $status -eq 0
-                    printf "%s may be found in the following packages:\n" "$argv[1]"
-                    printf "  %s\n" $__packages
-                else
-                    __fish_default_command_not_found_handler $argv[1]
-                end
-            end
-            # Use standard fish command not found handler otherwise
-        else
-            function __fish_command_not_found_handler --on-event fish_command_not_found
-                __fish_default_command_not_found_handler $argv[1]
-            end
-        end
+    # For backwards-compatibility - the event doesn't exist anymore so it's harmless.
+    function __fish_command_not_found_handler --on-event fish_command_not_found
```

Comment:
So peopla are calling  `__fish_command_not_found_handler` in their config?

Reply:
Not really, no.

This bit is really just about downgrading - prior to this PR fish had *no* handler without the function, so if one of those ran this __fish_config_interactive it would behave quite weirdly (it would just print the backtrace, without an actual error message).

It's not too important, tbh, but it's simple enough to add.